app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/localfile/LocalFileLabOrderSenderTest.groovy

```diff
@@ -56,6 +66,16 @@ class LocalFileLabOrderSenderTest extends Specification{
                     String getUnderlyingOrder() {
                         return " Second Mock String Order"
                     }
+
+                    @Override
+                    String getFhirResourceId() {
+                        return null
+                    }
+
+                    @Override
+                    String getPatientId() {
+                        return null
+                    }
                 }
```

Comment:
Maybe this can be converted to use the `OrdersMock` (or `LabOrderMock` if you take that suggestion) class that was made in this PR.